Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -479,6 +522,17 @@ namespace WhiteBox
         m_defaultModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "SketchMode");
         m_edgeRestoreModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "RestoreMode");
 
+        // temporary setting to disable this feature
+        if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get())
+        {
+            bool hasTransformMode = false;
+            settingsRegistry->Get(hasTransformMode, WhiteBoxTransformFeature);
+            if (hasTransformMode == true)
+            {
+                m_transformModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "TransformMode");
+            }
+        }
```

Comment:
question: By doing this, when the setting is 'off/disabled' is it impossible to get into this mode? We just want to make sure this feature is 'invisible' to users unless they have the setting on. It would be good to do some testing to make sure nothing blows up when it's disabled.